modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectRelationshipExtensionProvider.java

```diff
@@ -128,14 +138,27 @@ public Map<String, PropertyDefinition> getExtendedPropertyDefinitions(
 				continue;
 			}
 
+			if (Objects.equals(
```

Comment:
To be honest, I would move this method to a private method `_getPropertyType(objectDefinition, objectRelationship)` :thinking: as I think the flow of the method does not change with the if clause